MongoTrait/OptionalllyStatusable.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace OpenOrchestra\MongoTrait;
+
+use OpenOrchestra\ModelInterface\Model\StatusInterface;
+use OpenOrchestra\MongoTrait\IsStatusable;
+use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+
+/**
+ * Trait OptionalllyStatusable
+ */
+trait OptionalllyStatusable
```

Comment:
@alavieille effectively, or we should use optional status or optionally statusable. As all mongo traits are ended with -able, I choose the second alternative.